google/cloud/bigtable/table_admin.h

```diff
@@ -1115,10 +1121,14 @@ class TableAdmin {
   std::string project_id_;
   std::string instance_id_;
   std::string instance_name_;
+  //@{
+  /// These prototypes are only used as temporary storage during construction of
+  /// the class, where they are consolidated as common policies in `options_`.
```

Comment:
Can't we avoid doing that?

Reply:
Probably. But that is not something I am going to do in this PR.